source/_components/pilight.markdown

```diff
@@ -53,6 +53,11 @@ pilight:
       - 42
 ```
 
+## {% linkable_title Known issues %}
+
+- You always have to use the correct types for service data for the `pilight.send` service, as all data is sent unchecked and unchanged to the daemon. If you choose the wrong type, your call gets rejected and all you get is a `insufficient number of arguments` in the pilight log. To retrieve the right argument type, read the [implementation of your protocol](https://github.com/pilight/pilight/tree/master/libs/pilight/protocols).
```

Comment:
> is a insufficient number of arguments in the pilight log --> a Pilight send failed in the home assistant log
That's an generic error that does not point to the actual data type problem. The only hint that pilight got your data but rejected them is the log message in the pilight error log. It is necessary to point the user to the position where he can see that its not a connection problem but wrong argument types.